mainapp/tests.py

```diff
@@ -239,3 +239,64 @@ def test_read_book_status_code(self):
         response = self.client.get(self.url)
         self.assertEquals(response.status_code, 200)
         self.client.logout()
+
+
+class AddBooksTestCase(TestCase):
+    """Saved For Later Books View Test Case"""
+    def setUp(self):
+        self.client = Client()
+        self.user = User.objects.create_user(username='test_user', email='qwe@gmail.com')
+        self.user.set_password('foopassword')
+        self.user.save()
+
+    def test_save_book_status(self):
+        """Test the status code of save_book
+        When a book is Saved
+        """
+        book_id = 2
+        self.client.login(username='test_user', password='foopassword')
+
+        response = self.client.post(
+            reverse('save_book'),
+            data={'bookid': book_id},
+            HTTP_X_REQUESTED_WITH='XMLHttpRequest'
+        )
+        self.assertEquals(response.status_code, 200)
+        self.assertIn('true', response.content.decode("utf-8"))
+        self.client.logout()
+
+    def test_after_remove(self):
+        """Test the status code of
+        remove_saved_book When a book is removed
+        """
+        book_id = 2
```

Comment:
Would be nice if you fetch the book_id dynamically so the tests don't depend on the hardcoded value here.
The first book in the dataset would also do